asl-translator.cabal

```diff
@@ -133,6 +133,7 @@ test-suite asl-translator-formula-test
 executable asl-translator-concrete-test
   main-is: Concrete.hs
   build-depends:       base >= 4.10.0.0 && < 5,
+                       bv-sized >= 1.0.0,
```

Comment:
My usual upper bound comment

Reply:
Actually I think maybe this shouldn't have any bound at all, so it implicitly will use the library version?

Reply:
Oh, you are right - I was overzealous and didn't realize this was a test stanza

Reply:
I've removed the bound